Modules/_hashopenssl.c

```diff
@@ -1684,7 +1684,7 @@ _hashlib_HMAC_copy_impl(HMACobject *self)
         return _setException(PyExc_ValueError, NULL);
     }
 
-    retval = PyObject_New(HMACobject, Py_TYPE(self));
+    retval = (HMACobject *)PyObject_New(HMACobject, Py_TYPE(self));
```

Comment:
PyObject_New is macro which already does the cast, so the cast is superfluous actually:

https://github.com/python/cpython/blob/81a9b53fee79c4581bc1e2acae1c98ee0b692c7d/Include/objimpl.h#L130

Reply:
LOL our APIs are so mixed.